README.md

```diff
@@ -89,6 +90,22 @@ cd my-blog
 gem install
 ```
 
+### Test
+
+Test this project with:
+
+```
+cd my-blog
+```
+- MiniTest framework:
+```
+rails test
+```
+- RSpec framework:
+```
+rspec spec
+```
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 👥 Authors <a name="authors"></a>
```

Comment:
- [ ] In the Authors section, your LinkedIn link is broken :fearful: and giving 404 error :thinking: kindly fix it as suggested by the previous reviewer :+1: to make your readme professional :blush: